bigtable/examples/run_examples_utils.sh

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+readonly CBT_CMD="${CBT:-${GOPATH}/bin/cbt}"
+readonly CBT_EMULATOR_CMD="${CBT_EMULATOR:-${GOPATH}/bin/emulator}"
+
+# Run all the instance admin examples against production.
+#
+# This function allows us to keep a single place where all the examples are
+# listed. We want to run these examples in the continuous integration builds
+# because they rot otherwise.
+function run_all_instance_admin_examples() {
+  if [ ! -x ../examples/bigtable_samples_instance_admin ]; then
+    echo "Will not run the examples as the examples were not built"
+    return
+  fi
+
+  local project_id=$1
+  shift
+  local zone_id=$1
+  shift
+
+  # Create a (very likely unique) instance name.
+  local -r INSTANCE="in-$(date +%s)"
+
+  local ignore_unimplemented="/bin/false"
+  if [ ! -z "${BIGTABLE_EMULATOR_HOST:-}" ]; then
+    ignore_unimplemented="echo Ignoring error as the emulator does not implement this function"
+  fi
+
+  echo
+  echo "Run create-instance example."
+  ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}" || ${ignore_unimplemented}
+
+  echo
+  echo "Run list-instances example."
+  ../examples/bigtable_samples_instance_admin list-instances "${project_id}" || ${ignore_unimplemented}
+
+  echo
+  echo "Run get-instance example."
+  ../examples/bigtable_samples_instance_admin get-instance "${project_id}" "${INSTANCE}" || ${ignore_unimplemented}
+
+#  echo
+#  echo "Run list-clusters example."
```

Comment:
Is this test commented out because it is unimplemented? If so, care to add a TODO with a bug id here that matches the bug id in the TODO for the implementation?

Reply:
Fixed.